dev/create-release/release-util.sh

```diff
@@ -73,7 +73,9 @@ function fcreate_secure {
 }
 
 function check_for_tag {
-  curl -s --head --fail "$ASF_REPO_WEBUI;a=commit;h=$1" >/dev/null
+  # Check HTML body messages instead of header status codes. Apache GitBox returns
+  # a header with `200 OK` status code for both existing and non-existing tag URLs
```

Comment:
Right. GitHub repo can be used.
```bash
$ curl -s --head --fail "https://github.com/apache/spark/releases/tag/v2.4.0" > /dev/null
$ echo $?
0
$ curl -s --head --fail "https://github.com/apache/spark/releases/tag/v3.0.0" > /dev/null
$ echo $?
22
```

For me, since last bad experience, I'm pushing to only GitBox. So, I'm not sure whether `GitHub to GitBox` sync is safe or not in these days.

Reply:
GitHub to GitBox sync must be safe otherwise we have problems. At least I've updated my local `ASF_REPO` to github repo and merge PR to github repo directly.

Reply:
Then, shall I change it back to check header messages against GitHub tag urls? I can do that as a followup PR if you want.

Reply:
SGTM. I think it's a better idea to forget about GitBox. Once a committer has done the setup, he can pull/push to the github repo, which is the single south of truth.

Reply:
Yep. I'll make a PR soon. Thanks!

Reply:
Ur, are you thinking of changing all `gitbox` to `github` together?
Actually, we changed them from `git-wip-us` to `github` yesterday.
- https://github.com/apache/spark/pull/23454

For tag checking, it's okay to use GitHub only. But, I'm afraid if GitHub-only-usage is a potential violation of some Apache policies.

Reply:
Before proceeding that, let me search some from Apache websites.

Reply:
Good point on the apache policy concerns. For now I think we only need to use github at this particular place, because gitbox has a mistake about the reader status.

Reply:
Ya. It was just my confusion. My bad. It looks completely okay. 

- https://reference.apache.org/committer/github
> The ASF has a presence on GitHub at https://github.com/apache

- https://gitbox.apache.org/
> People that do not wish to utilize GitHub may continue using their ASF credentials to push code to gitbox.apache.org - we do not mandate use of one of the other.

Previously, `github` was able to be used at read-only pointers like `pom.xml`'s `connection`. After GitBox setup, it's okay for read/write pointers for all GitHub users.

For non-GitHub users (maybe BitBucket or GitLab users), we still use the `gitbox` URL at write-access-required pointer like `pom.xml`'s `developerConnection`.

Reply:
Yes. So for a `SPARK-26554` follow-up PR, I'll focus only this `check_for_tag` function at this time.

Reply:
@cloud-fan . I made a followup PR, https://github.com/apache/spark/pull/23482.